src/gtest/test_equihash.cpp

```diff
@@ -7,6 +7,7 @@
 
 #include "crypto/equihash.h"
 #include "uint256.h"
+#include "utilstrencodings.h"
```

Comment:
why do we need `utilstrencodings.h` here?

Reply:
It's needed for `ParseHex`.